src/modules/modCommands/commands/move.js

```diff
@@ -56,7 +56,7 @@ const move = async (message, limit, channel, after = null) => {
         );
```

Comment:
can we PLEASE also move the selected message?

Reply:
No i like it this way. Technically we could, but then I would need to convert this peice of code 
```js
msg = await message.channel.messages.fetch({ limit, after });
```
to something much more complicated, including fetching the selected message, concating the collections, and idk what else. This also would cause a super odd limit of 101 messages.

Reply:
Well then write the code for it yourself. Also outside the scope of this PR